src/components/ControlledForm/ControlledForm.stories.mdx

```diff
@@ -0,0 +1,107 @@
+# Form Component
+
+## Type: Simple
+
+## Form Props
+
+|     Prop      |                    Type                    |                                                                                                                                      Description                                                                                                                                       | Default | Mandatory |
+| :-----------: | :----------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :-------: |
+|    render     | (getInputControls, isValid, state) => void |                                                                                                                                                                                                                                 This props will render the Form fields                 |    -    |   false   |
```

Comment:
@higorquintao please add an observation explaining what each argument represents from the `render` property. Also based on your example below, it seems that these should be destructured values. Please represent this in the documentation.